### PR TITLE
fix: Remove use of global SkipEmptyRows

### PR DESCRIPTION
**Pull request by spangenberg**
All Tests and their state:
```
RESOURCES TestIntegrationDirectConnectGateways
RESOURCES TestIntegrationDirectconnectLags
RESOURCES TestIntegrationDirectconnectConnections
DONE TestIntegrationApigatewayVpcLinks
RESOURCES TestIntegrationApigatewayUsagePlans
RESOURCES TestIntegrationEfsFilesystems
FIXED TestIntegrationRoute53HostedZones
RESOURCES TestIntegrationEc2EbsSnapshots
RESOURCES TestIntegrationEc2VpnGateways
FIXED TestIntegrationEc2Vpcs
RESOURCES TestIntegrationEc2VpcPeeringConnections
RESOURCES TestIntegrationEc2VpcEndpoints
RESOURCES TestIntegrationEc2TransitGateways
FIXED TestIntegrationEc2Subnets
FIXED TestIntegrationEc2SecurityGroups
FIXED TestIntegrationEc2RouteTables
FIXED TestIntegrationEc2NetworkAcls
FIXED TestIntegrationEc2NatGateways
DONE TestIntegrationEc2InternetGateways
RESOURCES TestIntegrationEc2FlowLogs
RESOURCES TestIntegrationEc2CustomerGateways
FIXED TestIntegrationEc2EbsVolumes
RESOURCES TestIntegrationEc2Images
FIXED TestIntegrationEc2Instances
FIXED TestIntegrationEc2Eips
RESOURCES TestIntegrationApigatewayClientCertificates
RESOURCES TestIntegrationRoute53DelegationSet
RESOURCES TestIntegrationRoute53HealthChecks
RESOURCES TestIntegrationSageMakerModels
RESOURCES TestIntegrationSageMakerEndpointConfigurations
FIXED TestIntegrationSageMakerNotebookInstances
RESOURCES TestIntegrationApigatewayAPIKeys
FIXED TestIntegrationElasticsearchDomains
RESOURCES TestIntegrationApigatewayRestApis
RESOURCES TestIntegrationDirectconnectVirtualInterfaces
FIXED TestIntegrationRedshiftClusters
DONE TestIntegrationRedshiftSubnetGroups
FIXED TestIntegrationElbv2LoadBalancers
RESOURCES TestIntegrationElbv2TargetGroups
FIXED TestIntegrationRdsDbParameterGroups
FIXED TestIntegrationRdsClusterParameterGroups
FIXED TestIntegrationRdsDBSnapshots
RESOURCES TestIntegrationRdsClusterSnapshots
DONE TestIntegrationRdsInstances
DONE TestIntegrationRdsSubnetGroups
DONE TestIntegrationRdsClusters
DONE TestIntegrationRdsEventSubscriptions
FIXED TestIntegrationAutoscalingLaunchConfigurations
FIXED TestIntegrationAutoscalingGroups
RESOURCES TestIntegrationDmsReplicationInstances
RESOURCES TestIntegrationConfigConfigurationRecorders
RESOURCES TestIntegrationConfigConformancePack
RESOURCES TestIntegrationGuarddutyDetectors
FIXED TestIntegrationMQBrokers
RESOURCES TestIntegrationWAFWebACLs
DONE TestIntegrationApigatewayv2VpcLinks
FIXED TestIntegrationLambdaLayers
FIXED TestIntegrationLambdaFunctions
FXIED TestIntegrationWAFRuleGroups
FIXED TestIntegrationApigatewayv2ApisTest
RESOURCES TestIntegrationWAFRules
RESOURCES TestIntegrationEcrRepositories
DONE TestIntegrationS3Account
FIXED TestIntegrationEksClusters
FIXED TestIntegrationCloudtrailTrails
FIXED TestIntegrationS3Buckets
FIXED TestIntegrationIamUsers
DONE TestIntegrationIamSAMLIdentityProviders
RESOURCES TestIntegrationIamOpenidConnectIdentityProviders
FIXED TestIntegrationIamRoles
FIXED TestIntegrationIamPolicies
RESOURCES TestIntegrationIamGroups
RESOURCES TestIntegrationIamServerCertificates
RESOURCES TestIntegrationElbv1LoadBalancers
RESOURCES TestIntegrationElasticbeanstalkApplications
DONE TestIntegrationWAFv2ManagedRuleGroups
FIXED TestIntegrationSQSQueues
RESOURCES TestIntegrationWAFv2RuleGroups
RESOURCES TestIntegrationElasticbeanstalkEnvironments
FIXED TestIntegrationAccessAnalyzers
FIXED TestIntegrationEcsClusters
RESOURCES TestIntegrationEcsTaskDefinitions
RESOURCES TestIntegrationCloudwatchlogsFilters
FIXED TestIntegrationFsxBackups
FIXED TestIntegrationCloudfrontCachePolicies
FXIED TestIntegrationKmsKeys
DONE TestIntegrationSnsSubscriptions
DONE TestIntegrationSnsTopics
RESOURCES TestIntegrationWAFv2WebACLs
FIXED TestIntegrationEmrBlockPublicAccessConfigs
RESOURCES TestIntegrationCloudwatchAlarms
FIXED TestIntegrationEmrClusters
RESOURCES TestIntegrationCognitoIdentityPools
FIXED TestIntegrationCognitoUserPools
FIXED TestIntegrationCloudfrontDistributions
FIXED TestIntegrationSSMDocuments
RESOURCES TestIntegrationCodebuildProjects
RESOURCES TestIntegrationSecretsmanagerSecrets
```

Output from integration tests:
More information about running the tests [here](../docs/contributing/e2e_tests.md)

```
$ go test -tags=integration ./...
?   	github.com/cloudquery/cq-provider-aws	[no test files]
ok  	github.com/cloudquery/cq-provider-aws/client	(cached)
?   	github.com/cloudquery/cq-provider-aws/client/mocks	[no test files]
?   	github.com/cloudquery/cq-provider-aws/docs	[no test files]
?   	github.com/cloudquery/cq-provider-aws/resources/forks	[no test files]
?   	github.com/cloudquery/cq-provider-aws/resources/forks/apigatewayv2	[no test files]
?   	github.com/cloudquery/cq-provider-aws/resources/provider	[no test files]
ok  	github.com/cloudquery/cq-provider-aws/resources/services/accessanalyzer	(cached)
?   	github.com/cloudquery/cq-provider-aws/resources/services/acm	[no test files]
ok  	github.com/cloudquery/cq-provider-aws/resources/services/apigateway	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/apigatewayv2	(cached)
?   	github.com/cloudquery/cq-provider-aws/resources/services/applicationautoscaling	[no test files]
ok  	github.com/cloudquery/cq-provider-aws/resources/services/autoscaling	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/cloudfront	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/cloudtrail	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/cloudwatch	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/cloudwatchlogs	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/codebuild	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/cognito	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/config	(cached)
?   	github.com/cloudquery/cq-provider-aws/resources/services/dax	[no test files]
ok  	github.com/cloudquery/cq-provider-aws/resources/services/directconnect	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/dms	(cached)
?   	github.com/cloudquery/cq-provider-aws/resources/services/dynamodb	[no test files]
ok	github.com/cloudquery/cq-provider-aws/resources/services/ec2	149.435s
ok  	github.com/cloudquery/cq-provider-aws/resources/services/ecr	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/ecs	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/efs	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/eks	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/elasticbeanstalk	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/elasticsearch	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/elbv1	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/elbv2	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/emr	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/fsx	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/guardduty	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/iam	17.643s
ok  	github.com/cloudquery/cq-provider-aws/resources/services/kms	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/lambda	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/mq	(cached)
?   	github.com/cloudquery/cq-provider-aws/resources/services/organizations	[no test files]
ok  	github.com/cloudquery/cq-provider-aws/resources/services/rds	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/redshift	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/route53	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/s3	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/sagemaker	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/secretsmanager	(cached)
?   	github.com/cloudquery/cq-provider-aws/resources/services/sns	[no test files]
ok  	github.com/cloudquery/cq-provider-aws/resources/services/sqs	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/ssm	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/waf	(cached)
ok  	github.com/cloudquery/cq-provider-aws/resources/services/wafv2	(cached)
?   	github.com/cloudquery/cq-provider-aws/tools/endpoints	[no test files]
OK
```

### CODE REVIEW

File: resources/services/ec2/flow_logs.go [hunk 1]
```diff
@@ -7,18 +7,20 @@ import (
...
-		DeleteFilter: client.DeleteAccountRegionFilter,
-		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id"}},
+		Name:          "aws_ec2_flow_logs",
+		Description:   "Describes a flow log.",
+		Resolver:      fetchEc2FlowLogs,
+		Multiplex:     client.ServiceAccountRegionMultiplexer("ec2"),
+		IgnoreError:   client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter:  client.DeleteAccountRegionFilter,
+		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id"}},
+		IgnoreInTests: true,
```
Comment (yevgenypats) [inline resources/services/ec2/flow_logs.go:23]: we don't have a flow log in our test infrastructure?

File: resources/services/ec2/images.go [hunk 2]
```diff
@@ -7,18 +7,20 @@ import (
...
-		DeleteFilter: client.DeleteAccountRegionFilter,
-		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id"}},
+		Name:          "aws_ec2_images",
+		Description:   "Describes an image.",
+		Resolver:      fetchEc2Images,
+		Multiplex:     client.ServiceAccountRegionMultiplexer("ec2"),
+		IgnoreError:   client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter:  client.DeleteAccountRegionFilter,
+		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "id"}},
+		IgnoreInTests: true,
```
Comment (yevgenypats) [inline resources/services/ec2/images.go:23]: just double verifying if we don't have an image in our test infrastructure

**Review by yevgenypats — APPROVED** [review]
looks good. Just checking while testing you were running it against `cq-provider-aws` AWS account ?

File: resources/services/ec2/flow_logs.go (hunk 1 above)
Comment (spangenberg) [reply resources/services/ec2/flow_logs.go:23]: `    flow_logs_test.go:12: expected to have at least 1 entry at table aws_ec2_flow_logs got zero` if I remove it.

File: resources/services/ec2/images.go (hunk 2 above)
Comment (spangenberg) [reply resources/services/ec2/images.go:23]: `    images_test.go:12: expected to have at least 1 entry at table aws_ec2_images got zero` if I remove it.

**Comment by spangenberg** [comment]
> looks good. Just checking while testing you were running it against `cq-provider-aws` AWS account ?

Yep

**Comment by spangenberg** [comment]
Just rebased, no additional changes.

**Pull request merged**